torch/csrc/jit/script/compiler.cpp

```diff
@@ -1227,6 +1488,27 @@ struct to_ir {
       case TK_GE:
       case '*':
       case '/':
+      case '+':
```

Comment:
Why did these move into emitSugaredExpr? They are still always returning values, which was the purpose of the emitSimpleExpr node.

Reply:
Good point, I'll move them back into the emitSimpleExpr path